File: ivandrenc-cs50-problems-2020-x-sentimental-mario-more/mario.py
from cs50 import get_int
numero = get_int("Height: ")
while numero <= 0 or numero >8:
    numero = get_int("Height: ")
contador = 0
for i in range(numero):
    contador = contador + 1
    for j in range(numero - contador):
        print(" ", end="")
    for a in range(contador):
        print("#", end="")
    for b in range(2):
        print(" ", end="")
    for k in range(contador):
        print("#", end="")
    print()
        
        

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height = 0; //declaracion
    do
    {
        height = get_int("height: "); //pido la altura
    }
    while (height <= 0 || height > 8); //restriccion de numeros solo de 0-8



    int counter = 0; //counter para restar a los puntos cuando van bajando 

    for (int i = 0; i < height; i++)
    {
        counter = counter + 1;
        for (int j = height - counter; j > 0; j--) //aqui se hace por cada linia un punto menos
        {
            printf(" ");
        }

        for (int a = 1; a <= counter; a++) //aca se empieza desde un # hasta tener muchos como la variable height
        {
            printf("#");
        }
        printf("\n"); //salto de linia
    }


}

